statistics.cpp

```diff
@@ -36,14 +36,16 @@ double Statistics::log_likelihood(const VectorType& data,
     // further and see that instead of zero, a very small number relative to the rest of the
     // concentrations is perhaps more accurate and plays nicely with the logarithms used later.
     // To this end, we calculate the sum of all positive concentrations to get an idea of the
-    // size scale. Then in the pmf calculation, the negative values are changed to be 1e-9*norm.
```

Comment:
I think it was actually never the norm, right?

Reply:
It divided out, so I didn't include it in the calculation. If the concentration was 1e-9*norm, then the normalization step would be 1e-9*norm/norm=1e-9.